certified-connectors/USB4SAP/apidefinition.swagger.json

```diff
@@ -0,0 +1,70 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "USB4SAP",
+    "version": "1.0",
+    "description": "Universal Synch Bridge for SAP: One API for all SAP integration, analytics and UI needs. Simplify modern analytics, apps and automations. Provision SAP data, insights and automation On-demand.",
+    "contact":{
+      "name": "Ecoservity",
+      "url" : "https://www.ecoservity.com/#contact",
+      "email": "hello@ecoservity.com"
+
+    }
+  },
+
+  "host" : "http://35.208.99.120:8000",
+  "paths": {
+    "/sap/opu/odata/ECOS/OBJ2CLOUD_V2_SRV/ET_DatasetSet": {
+      "GET": {
+        "summary": "GET call to extract metadata & data from SAP data objects",
+        "operationId" : "GetSAPData",
+        "description": "The GET call extracts data along with its metadata from SAP, based on the selection criteria by user or background scheduled job",
+        "parameters": [
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "x-example": "ObjectName eq 'T001W' and ObjectType eq 'T' and Delimiter eq '|' and ImportGeneric eq 'ALVV-MG' and DataDest eq '0:0|' "
+          },
+          {
+            "name": "$format",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-example": "json"
+          }
+        ],
+        "responses": {
+          "200": { 
+            "description" : "OK",
+            "schema" : {
+              "$ref" : "#definitions/Message/"
```

Comment:
@ecsadm you haven't defined Message to reference it. Please refer to [coding standards](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters) if you want to use references or define a schema here. For more examples you can also see [connector samples](https://learn.microsoft.com/en-us/connectors/custom-connectors/samples).